frontend/packages/dev-console/src/components/helm/helm-utils.ts

```diff
@@ -66,6 +66,14 @@ export const filterHelmReleasesByName = (releases: HelmRelease[], filter: string
   return releases.filter((release: HelmRelease) => fuzzy(filter, release.name));
 };
 
+export const fetchHelmRelease = (
+  namespace: string,
+  helmReleaseName?: string,
+): Promise<HelmRelease> => {
+  const fetchString = `/api/helm/release?ns=${namespace}&release_name=${helmReleaseName}`;
```

Comment:
How does this differ from `/api/helm/releases?ns=${namespace}&release_name=${helmReleaseName}` ?

Reply:
`/api/helm/releases` returns list of all helm releases in that particular namespace, we don't accept `release_name` query param in this API, whereas to get particular release we have separate `GET /api/helm/release?namespace=${namespace}&release_name=${helmReleaseName}` which should be used

Reply:
Good to know. I asked because right below this method is `fetchHelmReleases` which accepts an optional `helmReleaseName`.
I think @rohitkrai03 may have added this by mistake then.
Could we clean that up at the same time?
That 2nd parameter doesn't seem to be used anywhere in the repo

Reply:
The idea was to have same function work for both list of helm releases and a specific release if release_name is provided. Jeff added this method when he was working on the release notes piece and I left it the way it is right now because I knew https://github.com/openshift/console/pull/4924 was going in soon.

Reply:
@akashshinde Once this PR - https://github.com/openshift/console/pull/4924 gets merged, UI will break with your changes. Then what is the need for this PR at the moment if you're still changing this to use older `release_name` query param? I mentioned this on the other PR as well.

Reply:
@rohitkrai03 My intention was to get this PR merged and then update `release_name` query param to `name` which would adhere to https://github.com/openshift/console/pull/4924. If you guys think this PR is not priority then I can make required UI changes to backend PR itself.

Reply:
Yeah, I think it would be better to add required UI changes to the backend PR itself. This API is not going to be affected by the backend change anyways.